app/views/tools/_title.html.haml

```diff
@@ -0,0 +1,5 @@
+- text_class = "text-dark" unless defined?(text_class)
+.row.no-gutters.title-container.title-strong
+  .col.bar.d-none.d-sm-block
+  .col.col-sm-auto.title{ class: text_class }= title
+  .col.bar.d-none.d-sm-block
```

Comment:
Recommend changing this file name to be `tools/_featured.html.haml` to better show the purpose of the file.

Reply:
It's actually just used as a `title` for both featured and non featured articles. So the purpose is actually correct IMO.

Reply:
Oh I see you are correct, this is the "Featured Articles" - "Other Articles" partial. I think it's fine to remove these headers in favor of simply having the styling be different. Also, while technically "articles" the Tools index should treat them more along the lines of tools to differentiate a bit more from the blog itself.